nixos/modules/rename.nix

```
@@ -295,6 +295,12 @@ in
     (mkRemovedOptionModule [ "services" "tedicross" ] ''
       The corresponding package was broken and removed from nixpkgs.
     '')
+    (mkRemovedOptionModule ["services" "rippled"] ''
+      The corresponding package was broken, abandoned upstream and thus removed from nixpkgs.
+    '')
+    (mkRemovedOptionModule ["services" "rippleDataApi"] ''
+      The corresponding package was broken, abandoned upstream and thus removed from nixpkgs.
+    '')
```

```suggestion
    (mkRemovedOptionModule [ "services" "rippled"] ''
      The corresponding package was broken, abandoned upstream and thus removed from nixpkgs.
    '')
    (mkRemovedOptionModule [ "services" "rippleDataApi"] ''
      The corresponding package was broken, abandoned upstream and thus removed from nixpkgs.
    '')
```

Otherwise nixfmt-rfc-style checker is unhappy